model/shared/src/main/scala/explore/model/PosAngle.scala

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats.Eq
+import cats.syntax.all._
+import lucuma.core.math.Angle
+
+sealed trait PosAngle {
+  def angle: Angle
+}
+
+object PosAngle {
+  case class Fixed(angle: Angle)               extends PosAngle {
+    override def toString: String = s"Fixed(${angle.toDoubleDegrees})"
+  }
+  case class AllowFlip(angle: Angle)           extends PosAngle {
+    override def toString: String = s"AllowFlip(${angle.toDoubleDegrees})"
+  }
+  case class AverageParallactic(angle: Angle)  extends PosAngle {
+    override def toString: String = s"AverageParallactic(${angle.toDoubleDegrees})"
+  }
```

Comment:
Should we be storing an `Angle` in the case of average parallactic?  It will depend on the time of day.

Reply:
<img width="927" alt="Screen Shot 2022-04-29 at 17 09 02" src="https://user-images.githubusercontent.com/4906023/166070595-a7db77f6-7579-4356-a429-de6c14c7ce2f.png">

Reply:
My understanding (which is vague) is that the angle in `AverageParallactic` will be calculated, that's why it is read only on the UI. I'm not yet clear how is it going to be calculated or the parameters used to do so